package/gluon-alt-esc-client/i18n/de.po

```diff
@@ -0,0 +1,49 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: PACKAGE VERSION\n"
+"PO-Revision-Date: 2017-11-13 08:00+0200\n"
+"Last-Translator:  Linus Lüssing <linus.luessing@c0d3.blue>\n"
+"Language-Team: German\n"
+"Language: de\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+
+msgid ""
+"Here you can add a WiFi interface with an alternative gateway for its "
+"Internet connectivity. Usually, you connect to a node which has the "
+"Alt-ESC-Provider package activated (although other systems can "
+"provide access too)."
+msgstr ""
+"Hier kannst du ein WiFi Interface mit einem alternativen Gateway für "
+"dessen Internetzugang hinzufügen. Normalerweise verbindest du dich mit "
+"einem Knoten, welcher das Alt-ESC-Provider Paket aktiviert hat (wobei "
+"auch andere Systeme Zugang bieten können)."
+
+msgid "Name (SSID)"
+msgstr "Name (SSID)"
+
+msgid "Example: \"Lisa's Garden Gate\""
+msgstr "Beispiel: \"Lisas Gartentor\""
```

Comment:
shouldn't we take an SSID that sounds like Freifunk as example? e.g. "Lisas Garten Freifunk"